Add 4 new questions to GitHub Foundations around GitHub administration based on Microsoft Learn guides and GitHub official documentation. (#209)
FidelusAleksander/ghcertified: content/questions/foundations/question-106.md

---
archetype: "questions"
title: "Question 106"

> question: "<What are the different levels of GitHub Administration hierarchy?  (Choose 3)>"

Reviewer: ```suggestion
question: "What are the different levels of GitHub organizational hierarchy?  (Choose three.)"
```

draft: false
---

> https://learn.microsoft.com/en-us/training/modules/github-introduction-administration/2-what-is-github-administration
- [x] Team level
- [x] Organization level
- [ ] Personal level

> > (optional) GitHub Personal is a type of account and not an organizational hierarchy on GitHub.

Reviewer: Hints are optional, but the word `optional` is not required there (or wanted)
```suggestion
> GitHub Personal is a type of account and not an organizational hierarchy on GitHub.
```

- [ ] Pro level

> > (optional) GitHub Pro is a type of account and not an organizational hierarchy on GitHub.

Reviewer: ```suggestion
> GitHub Pro is a type of account and not an organizational hierarchy on GitHub.
```

> - [x] Enterprize level

Reviewer 2: ```suggestion
- [x] Enterprise level
```

@FidelusAleksander  Oddly enough, the spelling test didn't catch this.

Reviewer: Indeed, quite odd :man_shrugging:

---
archetype: "questions"
title: "Question 106"

> question: "<What are the different levels of GitHub Administration hiearchy?  (Choose 3)>"

Reviewer 2: ```suggestion
question: "<What are the different levels of GitHub Administration hierarchy?  (Choose 3)>"
```
